### PR TITLE
fix: Recursion tutorial updates for README and package.json

### PR DESCRIPTION
**Pull request by barriebyron**
This PR fixes incorrect names in the README and the package.json file for the recursion tutorial

### CODE REVIEW

File: examples/zkapps/09-recursion/README.md
Comment (iregina) [inline examples/zkapps/09-recursion/README.md:1]: Good find! There's no such filed called main.js in /src

File: examples/zkapps/09-recursion/package.json
Comment (iregina) [inline examples/zkapps/09-recursion/package.json:1]: Nice catch up on the metadata / project name!

**Review by iregina — APPROVED** [review]
Looks good & straightforward. Can merge!

**Review by ymekuria — APPROVED** [review]
LGTM!

**Pull request merged**